src/components/ProgressComponent.vue

```diff
@@ -6,7 +6,7 @@
 				Congratulations on finishing the Orion camouflage grind! It's been a long ride! You first
 				started tracking your grind here
 				<b>{{ daysSinceStart }} days ago</b> on
-				{{ new Date(beganGrind).toLocaleDateString('en-US') }}.
+				{{ new Date(this.getBeganGrind).toLocaleDateString('en-US') }}.
```

Comment:
`this` shouldn't be needed here
```suggestion
				{{ new Date(getBeganGrind).toLocaleDateString('en-US') }}.
```